pytorch_lightning/trainer/connectors/logger_connector/result.py

```diff
@@ -604,6 +605,16 @@ def cpu(self) -> "ResultCollection":
         """Move all data to CPU."""
         return self.to(device="cpu")
 
+    def sync(self) -> None:
+        for result_metric in self.result_metrics:
+            if result_metric.is_tensor and not result_metric._is_synced:
```

Comment:
Same as in a previous comment

```suggestion
            if result_metric.is_tensor:
```

Reply:
Done !